modules/microshift-cni-multus-using-ipam.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift_multiple_networks/microshift-cni-multus-using.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="IP-address-management-types-and-additional-networks_{context}"]
+= IP address management types and additional networks
+
+IP addresses are provisioned for an additional network through an IP Address Management (IPAM) CNI plugin that you configure. Supported IP address provisioning types in {microshift-short} are `host-local`, `static`, and `dhcp`.
+
+[id="bridge-interface-specifics_{context}"]
+== bridge interface specifics
+When using `bridge` and the `dhcp` IPAM, a DHCP server listening on the bridged network is required. If you are using a firewall, configuring the firewalld service by running the `firewall-cmd --remove-service=dhcp` command to allow DHCP traffic on the network zone is also required.
+
+[id="macvlan-interface-specifics_{context}"]
+== macvlan interface specifics
+The `macvlan` type interface accesses the network that the host is connected to. This means that the interface can receive an IP address from the DHCP server on the host network if the `dhcp` IPAM plugin is used.
+
+[id="ipvlan-interface-specifics_{context}"]
+== ipvlan interface specifics
+The `ipvlan` interface also has direct access to the host network, but shares the MAC address with the host interface. The `ipvlan` type interface cannot be used with the `dhcp` plugin because of the shared MAC address. The IPAM plugin does not support the DHCP protocol with `ClientID`.
```

Comment:
We may need add all the CNI plugin types and different modes for each type in this doc:
[Microshift] Multus CNI bridge with host-local
[Microshift] Multus CNI bridge with static
[Microshift] Multus CNI bridge with dhcp
[Microshift] Multus CNI macvlan/bridge with host-local
[Microshift] Multus CNI macvlan/bridge with static
[Microshift] Multus CNI macvlan/private with host-local
[Microshift] Multus CNI macvlan/private with static
[Microshift] Multus CNI macvlan/vepa with host-local
[Microshift] Multus CNI macvlan/vepa with static

[Microshift] Multus CNI ipvlan/l2 with host-local
[Microshift] Multus CNI ipvlan/l2 with static
[Microshift] Multus CNI ipvlan/l3 with host-local
[Microshift] Multus CNI ipvlan/l3 with static

Reply:
@weliang1 @ShaunaDiaz in the main OCP documentation there is a list of CNI plugins for creating additional networks. That list links to more detail for each plugin. https://docs.openshift.com/container-platform/4.15/networking/multiple_networks/understanding-multiple-networks.html

In the enhancement proposal for Multus support with MicroShift 
https://github.com/openshift/enhancements/blob/master/enhancements/microshift/multus-cni-for-microshift.md the  "Goals" section calls out a subset of network plugins MicroShift will claim to support. I think this is where the list @weliang1 is suggesting comes from (i.e. IPAMs: host-local, dhcp, static and CNIs: bridge,macvlan, ipvlan)

Should we add a link to this OCP documentation rather than mentioning possible configuration types here?
Various network configurations can get complex and duplicating detail here could be hard to maintain.

Specifically these are the links I'm suggesting:
bridge additional network: https://docs.openshift.com/container-platform/4.15/networking/multiple_networks/configuring-additional-network.html#nw-multus-bridge-object_configuring-additional-network
IPVLAN additional network: https://docs.openshift.com/container-platform/4.15/networking/multiple_networks/configuring-additional-network.html#nw-multus-ipvlan-object_configuring-additional-network
MACVLAN additional network: https://docs.openshift.com/container-platform/4.15/networking/multiple_networks/configuring-additional-network.html#nw-multus-macvlan-object_configuring-additional-network

Reply:
These links were added here: https://github.com/openshift/openshift-docs/pull/74823. In this doc preview, they are here: https://75027--ocpdocs-pr.netlify.app/microshift/latest/microshift_networking/microshift_multiple_networks/microshift-cni-multus

Reply:
@weliang1 Could you discuss in the Slack thread and let us know your thoughts on this section? https://redhat-internal.slack.com/archives/C03CJTNLKAT/p1715175937495279